src/components/swapv2/styleds.tsx

```diff
@@ -368,13 +366,29 @@ export const LiveChartWrapper = styled.div`
   }
 `
 
-export const RoutesWrapper = styled(LiveChartWrapper)<{ isOpenChart: boolean }>`
-  margin-bottom: 30px;
+export const RoutesWrapper = styled(LiveChartWrapper)<{ isOpenChart: boolean; borderBottom?: boolean }>`
+  height: auto;
   margin-top: 4px;
-  max-height: ${({ isOpenChart }) => (isOpenChart ? '354px' : '700px')};
+  padding-bottom: 25px;
+  border-bottom: ${({ theme, borderBottom }) => (borderBottom ? `1px solid ${theme.border}` : 'none')};
+`
+
+export const TokenInfoWrapper = styled(LiveChartWrapper)`
+  display: block;
+  height: auto;
+  border-bottom: none;
+  @media only screen and (max-width: 768px) {
+    width: 100%;
+  }
+  @media screen and (min-width: 768px) and (max-width: 768px) {
```

Comment:
Can you use mediaWidth in theme object?

Reply:
I saw in the old code , there are many line code same as `@media only screen and (max-width: 768px)` ,... I copy.

Reply:
We should refactor this in the future, so for the new code, please use it 🥺.

Reply:
it was used like that before in this file, but changing to use mediaWidth is also a good practice

Reply:
got it.